### PR TITLE
Add v1 link

### PR DESCRIPTION
**Pull request by Seroxdesign**
Fixes #

**Changes proposed in this pull request:**

- Added Link button to mobile menu
- Added Link button to desktop invest nav
- Link currently takes no where because we haven't changed versions yet

### CODE REVIEW

**Comment by josealonso** [comment]
@Seroxdesign, glad to see you used the *i18n* file.

File: src/components/layout/InvestNav/index.tsx [hunk 1]
```diff
@@ -108,6 +109,15 @@ export const InvestNav = () => {
         <div className={styles.nav_text_tutorial}>{t('Tutorial')}</div>
         <FontIcon name={FontIconName.External} size={16} />
       </NavLink>
+
+      <NavLink
+        to={{ pathname: RICOCHET_V1_LINK }}
+        className={styles.nav_link_desktop_only}
+        activeClassName={styles.nav_link_active}
+      >
+        <div className={styles.nav_text_tutorial}>{t('Ricochet V1')}</div>
```
Comment (zhyd1997) [inline src/components/layout/InvestNav/index.tsx:118]: Before using `i18n`, we should declare it in [the config file](https://github.com/Ricochet-Exchange/ricochet-frontend/blob/8659efda50deb3f328d9c2f51a6ae6d2a9809be2/src/i18n/en/main.json) first.

File: src/containers/main/UpgradeContainer/index.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ import { queryFlows } from '../../../api';
 import { Flow } from '../../../types/flow';
 
 function getFormattedNumber(num: string) {
-  return parseFloat(num).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  return parseFloat(num).toString().replace(/\B(?=(\d{3})+(?!\d))/g, '');
```
Comment (elvijsTDL) [inline src/containers/main/UpgradeContainer/index.tsx:26]: Why this change? Doesn't seem to be link related

**Review by elvijsTDL — CHANGES_REQUESTED** [review]

File: src/containers/main/UpgradeContainer/index.tsx (hunk 2 above)
Comment (Seroxdesign) [reply src/containers/main/UpgradeContainer/index.tsx:26]: https://github.com/Ricochet-Exchange/ricochet-frontend/issues/493

**Review by mikeghen — APPROVED** [review]
